src/content/admin/cleanup.mdx

```diff
@@ -20,15 +20,26 @@ Learn how to configure this in the [configuration steps](#configuration-steps-).
 
 Okteto considers an application or resources inactive if a user hasn't performed any of the following tasks during the `inactivity` period:
 
-- Upgrade or redeploy via the UI or the command line.
-- Launched a development container with `okteto up`.
-- Deploy the latest version of your code with `okteto deploy`.
+- Upgrade or redeploy via the UI or the command line
+- Launched a development container with `okteto up`
+- Deploy the latest version of your code with `okteto deploy`
 
 Service accounts, config maps, secrets, or volumes you create will be unaffected by this operation.
 
-## Wake sleeping resources
+### Events That Count as Activity to Reset the Sleep Counter
 
-Okteto's UI will notify you when there are sleeping resources in your namespace. Press the **Wake all** button to activate all sleeping resources at once.
+The following events will reset the inactivity counter, preventing the resources from entering a sleep state:
+
+- Deploying a Kubernetes Deployment or StatefulSet with `kubectl` or similar tools
+- Running `okteto deploy`
+- Running `okteto up`
+- Maintaining an active `okteto up` session. If the `upSessionByLastSyncedFile` setting is enabled, this only counts if files are actively being synced (based on the last synced file)
+
+## Wake Sleeping Resources
```

Comment:
I know it's not part of this change, but we have a sample screenshot for how to wake resources, but not for how to sleep them with the UI. Add perhaps?

Reply:
Good callout. We're also missing information on how to manually sleep a resource from the namespace list view. Will add.